### PR TITLE
[Backport 2.x] Exclude jettison version brought in with hadoop-minicluster

### PR DESCRIPTION
**Pull request by ryanbogan**
Signed-off-by: Ryan Bogan <rbogan@amazon.com>

### Description
Backports [4787](https://github.com/opensearch-project/OpenSearch/pull/4787) to 2.x

### Check List
- [X ] New functionality includes testing.
  - [X ] All tests pass
- [ X] New functionality has been documented.
  - [X ] New functionality has javadoc added
- [X ] Commits are signed per the DCO using --signoff
- [ X] Commit changes are listed out in CHANGELOG.md file (See: [Changelog](../blob/main/CONTRIBUTING.md#changelog))

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,9 +19,37 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Introduce experimental searchable snapshot API ([#4680](https://github.com/opensearch-project/OpenSearch/pull/4680))
 
 ### Dependencies
+<<<<<<< HEAD
```
Comment (reta) [inline CHANGELOG.md:22]: Conflicts?

**Review by reta — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ryanbogan) [reply CHANGELOG.md]: Should be fixed now

File: CHANGELOG.md [hunk 2]
```diff
@@ -22,6 +22,32 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
...
+- Bumps `reactor-netty-core` from 1.0.19 to 1.0.22 ([#4447](https://github.com/opensearch-project/OpenSearch/pull/4447))
+- Bumps `reactive-streams` from 1.0.3 to 1.0.4 ([#4488](https://github.com/opensearch-project/OpenSearch/pull/4488))
+- Bumps `com.diffplug.spotless` from 6.10.0 to 6.11.0 ([#4547](https://github.com/opensearch-project/OpenSearch/pull/4547))
+- Bumps `reactor-core` from 3.4.18 to 3.4.23 ([#4548](https://github.com/opensearch-project/OpenSearch/pull/4548))
+- Bumps `jempbox` from 1.8.16 to 1.8.17 ([#4550](https://github.com/opensearch-project/OpenSearch/pull/4550))
+- Bumps `hadoop-hdfs` from 3.3.3 to 3.3.4 ([#4644](https://github.com/opensearch-project/OpenSearch/pull/4644))
+- Bumps `jna` from 5.11.0 to 5.12.1 ([#4656](https://github.com/opensearch-project/OpenSearch/pull/4656))
+- Update Jackson Databind to 2.13.4.2 (addressing CVE-2022-42003) ([#4779](https://github.com/opensearch-project/OpenSearch/pull/4779))
+- Bumps `tika` from 2.4.0 to 2.5.0 ([#4791](https://github.com/opensearch-project/OpenSearch/pull/4791))
+- Exclude jettison version brought in with hadoop-minicluster. ([#4787](https://github.com/opensearch-project/OpenSearch/pull/4787))
```
[the remark this answers is not in the document]
Comment (ryanbogan) [reply CHANGELOG.md:49]: Yeah you're right

[the remark this answers is not in the document]
Comment (ryanbogan) [reply CHANGELOG.md:25]: Just fixed it

File: CHANGELOG.md [hunk 3]
```diff
@@ -22,6 +22,8 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Bumps `com.diffplug.spotless` from 6.9.1 to 6.10.0
 - Bumps `xmlbeans` from 5.1.0 to 5.1.1
 - Bumps `hadoop-hdfs` from 3.3.3 to 3.3.4
+- Exclude jettison version brought in with hadoop-minicluster. ([#4787](https://github.com/opensearch-project/OpenSearch/pull/4787))
+
```
Comment (andrross) [inline CHANGELOG.md:26]: Nitpick, but the original PR doesn't add this extra line so probably better not to add it here to minimize conflicts.

Comment (ryanbogan) [reply CHANGELOG.md:26]: Sure

**Comment by ryanbogan** [comment]
@reta All requested changes have been addressed.  Can you review again?

**Review by dblock — APPROVED** [review]

**Review by dblock — APPROVED** [review]

**Pull request merged**